FreeRTOS-Plus/Demo/AWS/Device_Shadow_Windows_Simulator/Common/main.c

```diff
@@ -26,7 +26,7 @@
  */
 
 /***
- * See https://www.FreeRTOS.org/mqtt_lts/index.html for configuration and usage instructions.
+ * See https://www.FreeRTOS.org/coremqtt for configuration and usage instructions.
```

Comment:
Was this supposed to be suffixed with `*/index.html` ?

Reply:
The default behaviour of the web browser is to try and open index.html - so its not necessary to add it manually.